### PR TITLE
implemented sigar plugin

### PR DESCRIPTION
**Pull request by seut**

### CODE REVIEW

File: core/src/main/java/io/crate/plugin/PluginLoader.java [hunk 1]
```diff
@@ -183,8 +183,9 @@ public String apply(Plugin input) {
                             checkJarHell(libURL);
                             pluginUrls.add(libURL);
                         } catch (Exception e) {
-                            logger.warn(
-                                    "Library " + libURL + " of plugin " + pluginURL + " already loaded, will ignore");
+                            String msg = "Library " + libURL + " of plugin " + pluginURL + " already loaded";
+                            logger.error(msg, e);
```
Comment (dobe) [inline core/src/main/java/io/crate/plugin/PluginLoader.java]: while you are at it you could also use variables here like in the other logger calls

File: core/src/main/java/io/crate/plugin/PluginLoader.java [hunk 2]
```diff
@@ -183,8 +183,9 @@ public String apply(Plugin input) {
...
+                            throw new RuntimeException(e);
```
Comment (dobe) [inline core/src/main/java/io/crate/plugin/PluginLoader.java]: do we need to wrapt this exception, why not throwing e directly?

Comment (seut) [inline core/src/main/java/io/crate/plugin/PluginLoader.java]: must be a runtime exception to be throwable

Comment (dobe) [inline core/src/main/java/io/crate/plugin/PluginLoader.java]: you can use Throwables.propagate

File: core/src/main/java/io/crate/plugin/PluginLoader.java [hunk 3]
```diff
@@ -183,13 +185,15 @@ public String apply(Plugin input) {
                             checkJarHell(libURL);
                             pluginUrls.add(libURL);
                         } catch (Exception e) {
-                            logger.warn(
-                                    "Library " + libURL + " of plugin " + pluginURL + " already loaded, will ignore");
+                            String msg = String.format(Locale.ENGLISH,
+                                    "Library %s of plugin %s already loaded", libURL, pluginURL);
```
Comment (dobe) [inline core/src/main/java/io/crate/plugin/PluginLoader.java]: this is built in in the logger. use logger.warn("Libary {} ...", ...)

Comment (seut) [inline core/src/main/java/io/crate/plugin/PluginLoader.java]: i know, but message should appear in exception also

**Comment by dobe** [comment]
ack

**Pull request merged**